src/vm.rs

```diff
@@ -95,11 +97,13 @@ pub trait InstructionMeter {
     /// Get the number of remaining instructions allowed
     fn get_remaining(&self) -> u64;
 }
-struct DefaultInstructionMeter {}
+
+/// Instruction meter without a limit
+pub struct DefaultInstructionMeter {}
 impl InstructionMeter for DefaultInstructionMeter {
     fn consume(&mut self, _amount: u64) {}
     fn get_remaining(&self) -> u64 {
-        std::u64::MAX
+        std::i64::MAX as u64
```

Comment:
Why?

Reply:
I guess I should add a comment in the source code, how the instruction meter for the JIT works.
It requires a signed integer because it adds and subtracts around the initial count (which represents 0).
Thus, it makes sense to only use 63 bit for the default meter to have an equal distance in both directions (- and +).